plugins/inputs/hystrix_stream/stream_parser.go

```diff
@@ -0,0 +1,183 @@
+	} `json:"latencyExecute"`
+	LatencyTotalMean int `json:"latencyTotal_mean"`
+	LatencyTotal     struct {
+		Num0   int `json:"0"`
+		Num25  int `json:"25"`
+		Num50  int `json:"50"`
+		Num75  int `json:"75"`
+		Num90  int `json:"90"`
+		Num95  int `json:"95"`
+		Num99  int `json:"99"`
+		Num100 int `json:"100"`
+		Nine95 int `json:"99.5"`
+	} `json:"latencyTotal"`
+	PropertyValueCircuitBreakerRequestVolumeThreshold             int         `json:"propertyValue_circuitBreakerRequestVolumeThreshold"`
+	PropertyValueCircuitBreakerSleepWindowInMilliseconds          int         `json:"propertyValue_circuitBreakerSleepWindowInMilliseconds"`
+	PropertyValueCircuitBreakerErrorThresholdPercentage           int         `json:"propertyValue_circuitBreakerErrorThresholdPercentage"`
+	PropertyValueCircuitBreakerForceOpen                          bool        `json:"propertyValue_circuitBreakerForceOpen"`
+	PropertyValueCircuitBreakerForceClosed                        bool        `json:"propertyValue_circuitBreakerForceClosed"`
+	PropertyValueCircuitBreakerEnabled                            bool        `json:"propertyValue_circuitBreakerEnabled"`
+	PropertyValueExecutionIsolationStrategy                       string      `json:"propertyValue_executionIsolationStrategy"`
+	PropertyValueExecutionIsolationThreadTimeoutInMilliseconds    int         `json:"propertyValue_executionIsolationThreadTimeoutInMilliseconds"`
+	PropertyValueExecutionTimeoutInMilliseconds                   int         `json:"propertyValue_executionTimeoutInMilliseconds"`
+	PropertyValueExecutionIsolationThreadInterruptOnTimeout       bool        `json:"propertyValue_executionIsolationThreadInterruptOnTimeout"`
+	PropertyValueExecutionIsolationThreadPoolKeyOverride          interface{} `json:"propertyValue_executionIsolationThreadPoolKeyOverride"`
+	PropertyValueExecutionIsolationSemaphoreMaxConcurrentRequests int         `json:"propertyValue_executionIsolationSemaphoreMaxConcurrentRequests"`
+	PropertyValueFallbackIsolationSemaphoreMaxConcurrentRequests  int         `json:"propertyValue_fallbackIsolationSemaphoreMaxConcurrentRequests"`
+	PropertyValueMetricsRollingStatisticalWindowInMilliseconds    int         `json:"propertyValue_metricsRollingStatisticalWindowInMilliseconds"`
+	PropertyValueRequestCacheEnabled                              bool        `json:"propertyValue_requestCacheEnabled"`
+	PropertyValueRequestLogEnabled                                bool        `json:"propertyValue_requestLogEnabled"`
+	ReportingHosts                                                int         `json:"reportingHosts"`
+	ThreadPool                                                    string      `json:"threadPool"`
+}
+
+var (
+	healthy       = false
+	scanner       *bufio.Scanner
+	cachedEntries []HystrixStreamEntry
+	reader        io.ReadCloser
+	cacheLock     sync.Mutex
+)
+
+func latestEntries(url string) ([]HystrixStreamEntry, error) {
+
+	if !healthy {
+		resp, err := http.Get(url)
```

Comment:
Since we are doing HTTP, we will need to add support for a timeout, TLS, and use a custom `http.Client`.  The [http input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/http) is probably the simplest example.